cmd/project.go

```diff
@@ -72,7 +72,17 @@ func (p *Project) createLicenseFile() error {
 	data := map[string]interface{}{
 		"copyright": copyrightLine(),
 	}
-	licenseFile, err := os.Create(fmt.Sprintf("%s/LICENSE", p.AbsolutePath))
+
+	licensePath := p.AbsolutePath + "/LICENSE"
```

Comment:
This is nicer, but the rest of the code use the previous style so this create inconsistency.

It would be better to keep the current style for this change, and if changing the way paths are created is needed, do this in the entire file or project.